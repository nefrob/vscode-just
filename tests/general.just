# Comments

# I am a comment
