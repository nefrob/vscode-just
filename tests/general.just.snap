># Comments
#^^^^^^^^^^ source.just comment.line.number-sign.just
>
># I am a comment
#^^^^^^^^^^^^^^^^ source.just comment.line.number-sign.just
>